chapters/VK_QNX_external_memory_screen_buffer/device_memory.adoc

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2023 QNX Software Systems
```

Comment:
It's up to you, but '(c)' literally means 'Copyright' and has not been required since 1989, so could be removed.